deployment/helm/node-feature-discovery/templates/worker.yaml

```diff
@@ -47,9 +47,38 @@ spec:
         image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
         imagePullPolicy: {{ .Values.image.pullPolicy }}
         livenessProbe:
-          {{- toYaml .Values.worker.livenessProbe | nindent 12 }}
+          grpc:
+            port: {{ .Values.worker.healthPort | default "8082" }}
+        {{- with .Values.worker.livenessProbe.initialDelaySeconds }}
+          initialDelaySeconds: {{ . }}
+        {{- end }}
+        {{- with .Values.worker.livenessProbe.failureThreshold }}
+          failureThreshold: {{ . }}
+        {{- end }}
+        {{- with .Values.worker.livenessProbe.periodSeconds }}
+          periodSeconds: {{ . }}
+        {{- end }}
+        {{- with .Values.worker.livenessProbe.timeoutSeconds }}
+          timeoutSeconds: {{ . }}
+        {{- end }}
         readinessProbe:
-          {{- toYaml .Values.worker.readinessProbe | nindent 12 }}
+          grpc:
+            port: {{ .Values.worker.healthPort | default "8082" }}
+        {{- with .Values.worker.readinessProbe.initialDelaySeconds }}
+          initialDelaySeconds: {{ . }}
+        {{- end }}
+        {{- with .Values.worker.readinessProbe.failureThreshold }}
+          failureThreshold: {{ . }}
+        {{- end }}
+        {{- with .Values.worker.readinessProbe.periodSeconds }}
+          periodSeconds: {{ . }}
+        {{- end }}
+        {{- with .Values.worker.readinessProbe.timeoutSeconds }}
```

Comment:
this works when timeoutSecond is not provided in values? just making sure :)

Reply:
it works, yes. Currently there are only comments in the values.yaml :)

Reply:
Given the NV-heavy presence on this PR, I'll leave the final decision to @marquiz